tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py

```diff
@@ -109,3 +109,122 @@ def test002_create_vms_with_correct_params(self):
+        self.wait_for_service_action_status(self.vm1, res[self.vm1]['portforward_create'])
+
+        self.log(" Check that portforward created, should succeed.")
+        self.assertTrue(self.get_portforward_list(self.cs1, self.vm1))
+
+        self.log("Delete the portforward created, should succeed")
+        temp_actions = {'vdc': {'actions': ['portforward_delete'], 'service': self.vm1, 
+                        'args': {'ports': {'source': public_port, 'destination': local_port}}}}
+        res = self.create_vm(accounts=self.accounts, cloudspaces=self.cloudspaces,
+                             vms=self.vms, temp_actions=temp_actions)
+        self.wait_for_service_action_status(self.vm1, res[self.vm1]['portforward_delete'])
+
+        self.log('Check that the portforward  deleted from cloudspace [CS1] successfully , should succeed.')
+        self.assertFalse(self.get_portforward_list(self.cs1, self.vm1))
+
+    def test002_start_stop_vm(self):
+        """ ZRT-OVC-013
+        *Test case for testing start and stop vm .*
+
+        **Test Scenario:**
+
+        #. Create a vm[vm1], should succeed.
+        #. Stop [VM1], should succceed.
+        #. Check that [VM1] halted.
+        #. Strat [VM1], should succeed.
+        #. Check that [VM1] running.
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.log("Stop [VM1], should succceed.")
+        temp_actions = {'node': {'actions': ['stop'], 'service': self.vm1}}
+        res = self.create_vm(accounts=self.accounts, cloudspaces=self.cloudspaces,
+                             vms=self.vms, temp_actions=temp_actions)
+        self.wait_for_service_action_status(self.vm1, res[self.vm1]['stop'])
+
+        self.log(" Check that [VM1] halted.")
+        cloudspaceId = self.get_cloudspace(self.cs1)['id']
+        vm = self.get_vm(cloudspaceId, self.vm1)
+        self.assertEqual(vm["status"], "HALTED")
+
+        self.log("Start [VM1], should succceed.")     
+        temp_actions = {'node': {'actions': ['start'], 'service': self.vm1}}
+        res = self.create_vm(accounts=self.accounts, cloudspaces=self.cloudspaces,
+                             vms=self.vms, temp_actions=temp_actions)
+        self.wait_for_service_action_status(self.vm1, res[self.vm1]['start'])
+
+        self.log(" Check that [VM1] running.")
+        cloudspaceId = self.get_cloudspace(self.cs1)['id']
+        vm = self.get_vm(cloudspaceId, self.vm1)
+        self.assertEqual(vm["status"], "RUNNING")
+
+    @classmethod
+    def tearDownClass(cls):
+        self = cls()
+        temp_actions = {'account': {'actions': ['uninstall']}}
+        if self.check_if_service_exist(self.acc1):
+            res = self.create_account(openvcloud=self.openvcloud, vdcusers=self.vdcusers,
+                                    accounts=self.accounts, temp_actions=temp_actions)
+            self.wait_for_service_action_status(self.acc1, res[self.acc1]['uninstall'])
```

Comment:
decrease the timeout to 20 for example in case the the env hanged .. also u don't need than 10 to 20 secs till the blueprint is excuted